roles/business.md

```diff
@@ -2,6 +2,22 @@
 
 ## Business Traits
 
+### +Academy
+
+We need someone:
+
+- to prepare and deliver the weekly learning programmes
+- to provide appropriate feedback and coaching, to help academy members to succeed  
+- to use a variety of teaching methods to challenge and engage academy members
+- to be the PO for the greenfield project
+
+We expect someone in the role:
+
+- to be autonomous in leading the academy
+- to be well organised and committed
```

Comment:
I see this as above and beyond. e.g.

- You need to be much more organised to support 4X academy members turning up each morning, who are expecting to learn a load of new stuff. Preparation needs to be done ahead of time, clear plans for the day need to be sorted etc. I see this being quite a different approach to what we do now, which is why i've highlighted. 
  
- Regularly turning up late (which a few of us do) is a problem if you're running the academy. We've also got more stringent requirements around holiday taking, as important for consistency. Takes a certain amount of commitment to sacrifice some things, in order to run this.